### DISCUSSION TITLE
Failed to Obtain Access Token with IDS 6.0

### DISCUSSION THREAD

**Discussion by cbdg-mitch** [requarks/wiki#5198] [Error / Bug Report]
Greetings everyone,

I'm trying to get Wiki.Js working with my Identity Server 6.0 instance. The IDS works great with all of my other apps, and I used the Well Known configuration for all of the values in the wiki config. When I visit my wiki, it does properly redirect to the IDS login page, and I can see in the IDS console that the login worked and the token was granted, but then Wiki just pops up with the oops screen.

Any ideas here? What other information can I provide to help troubleshoot?

"ClientId": ".. removed",
"ClientName": ".. removed",
"RedirectUri": "https://removed.com/login/bcc9c542-0598-4d41-acd6-00edecbe2d75/callback",
"Endpoint": "Authorize",
"SubjectId": "6b14edd9-e399-ea11-86e9-000d3a11483c",
"Scopes": "openid profile email",
"GrantType": "authorization_code",
"Tokens": [
{
"TokenType": "code",
"TokenValue": "****C2-1"
}
],
"Category": "Token",
"Name": "Token Issued Success",
"EventType": "Success",
"Id": 2000,
"ActivityId": "0HMGOUEHIGSNU:00000043",
"TimeStamp": "2022-04-08T00:07:15Z",
"ProcessId": 14720,
"LocalIpAddress": "::1:44309",
"RemoteIpAddress": "::1"

**Comment by cbdg-mitch** [comment]
@NGPixel? Anyone?

**Comment by NGPixel** [comment]
Please don't open duplicate tickets.

As for your problem, I really don't know.
